incubator/tensorflow-inception/README.md

```diff
@@ -31,8 +31,9 @@ The following table lists the configurable parameters of the TensorFlow inceptio
 | `ImageTag`       | Container image tag                | `latest`                                                          |
 | `Replicas`       | k8s deployment replicas            | `1`                                                               |
 | `Component`      | k8s selector key                   | `tensorflow-inception`                                            |
-| `Cpu`            | container requested cpu            | `100m`                                                            |
-| `Memory`         | container requested memory         | `512Mi`                                                           |
+| `Resources.limits.cpu`            | container requested cpu            | `100m`                                                            |
```

Comment:
If you use just a `resources` here with default `{}`, people will know what to do and understand and you don't have to differentiate in the docs between requests and limits.